### PR TITLE
Pine Forest map by mzf53

### PR DESCRIPTION
**Pull request by mzf53**

### CODE REVIEW

**Comment by Avyukt1** [comment]
Please add a screenshot. Press F12 while flying around your map.

**Comment by olliy1or** [comment]
> Please add a screenshot. Press F12 while flying around your map.

ther

**Comment by Avyukt1** [comment]
> > Please add a screenshot. Press F12 while flying around your map.
> 
> there is a screenshot

I didn't see it last night.

**Comment by xenonca** [comment]
Review Required label needed I guess @olliy1or

**Comment by olliy1or** [comment]
oh ye

**Comment by JostP** [comment]
Forest maps are not very popular in CTF. We had a maps popularity vote and forest maps got very bad votes, some of them were even removed. I don't think this map will be added.

**Review by olliy1or — DISMISSED** [review]

File: pine_forest/map.conf [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+team.1 = red
+team.1.pos = (53,-31,97)
+team.2.pos = (-82,-35,-70)
+team.2.color = blue
+author = mzf53
+rotation = z
+h = 140
+team.2 = blue
+name = Pine Forest
+initial_stuff = default:pick_stone,default:torch 5
```
Comment (JostP) [inline pine_forest/map.conf:11]:
```suggestion
initial_stuff = default:pick_stone,default:torch 10
```

**Comment by JostP** [comment]
We decided to add it, you just have to change screenshot. Use this one:
screenshot

**Review by JostP — APPROVED** [review]

**Pull request merged**